lisa/util/process.py

```diff
@@ -114,7 +114,7 @@ def start(
                 cwd=cwd_path,
                 update_env=new_envs,
                 allow_error=True,
-                store_pid=True,
+                store_pid=self._is_posix,
```

Comment:
What Windows version is used here? We can support only Windows 10 and latest Windows Server versions.

Reply:
10